<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataTemplateOptionsDialog</class>
 <widget class="QDialog" name="MetadataTemplateOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metadata template options</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_metadata_file_options">
     <property name="title">
      <string>Metadata file options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="findByPositionRadioButton">
        <property name="text">
         <string>find parameter by position (be careful if you use one metadata file for different data files)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="findByNameOrAbbreviationRadioButton">
        <property name="text">
         <string>find parameter by name or abbreviation (a parameter database will be used)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="findByNameOnlyRadioButton">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>find parameter by name only (a parameter database will be used)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_metadata_template_options">
     <property name="title">
      <string>Create metadata template options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="writeParameterImportFileCheckBox">
        <property name="text">
         <string>write parameter import file. After completing &quot;imp_Parameter.txt&quot; create an issue (https://issues.pangaea.de).</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="addSpeciesNamesCheckBox">
        <property name="text">
         <string>add column with species names to parameter import file for WoRMS matching</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findByPositionRadioButton</tabstop>
  <tabstop>findByNameOrAbbreviationRadioButton</tabstop>
  <tabstop>findByNameOnlyRadioButton</tabstop>
  <tabstop>writeParameterImportFileCheckBox</tabstop>
  <tabstop>addSpeciesNamesCheckBox</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
